ddtrace/context.py

```diff
@@ -125,9 +131,63 @@ def _traceparent(self):
         # type: () -> str
         if self.trace_id is None or self.span_id is None:
```

Comment:
Since we can now get the trace id from self._meta do we need to update this if statement?

Reply:
Oh I wasn't aware of that, why do we ever put trace id in meta?